tests/trx_finality_status_forked_test.py

```diff
@@ -121,12 +121,20 @@ def getState(status):
 
     def getBlockNum(status):
         assert status is not None, "ERROR: getTransactionStatus failed to return any status"
-        assert "head_number" in status, \
-            f"ERROR: getTransactionStatus returned a status object that didn't have a \"head_number\" field. state: {json.dumps(status, indent=1)}"
         if "block_number" in status:
             return status["block_number"]
+        assert "head_number" in status, \
+            f"ERROR: getTransactionStatus returned a status object that didn't have a \"head_number\" field. state: {json.dumps(status, indent=1)}"
```

Comment:
It might be better `state:` to be `status:` to match `status object` just a few words before,